node/src/service.rs

```diff
@@ -221,20 +230,37 @@ where
 		telemetry
 	});
 
+	let registry = config.prometheus_registry();
+
 	let transaction_pool = sc_transaction_pool::BasicPool::new_full(
 		config.transaction_pool.clone(),
 		config.role.is_authority().into(),
-		config.prometheus_registry(),
+		registry,
 		task_manager.spawn_essential_handle(),
 		client.clone(),
 	);
 
-	let import_queue = parachain_build_import_queue::<RuntimeApi, Executor>(
-		client.clone(),
-		config,
-		telemetry.as_ref().map(|telemetry| telemetry.handle()),
-		&task_manager,
-	)?;
+	let select_chain = if instant_seal {
+		Some(LongestChain::new(backend.clone()))
+	} else {
+		None
+	};
+
+	let import_queue = if instant_seal {
+		// instance sealing
```

Comment:
instant sealing?

Reply:
it's an alternative block production (and finality IIRC) meant for local testnets. you can run a testnet and build a block on every transaction. 
I'm also curious where this is coming from, though, cause it doesn't seem to be anywhere in the parachain template, yet?

Reply:
we dont use it for running any chain, but it is used for execution of some commands... i am not sure why tho. @Roznovjak

Reply:
This part was inspired by some other projects.

Reply:
i would then either utilize it correctly or remove it completely

Reply:
removed